lib/solveEquation/stepThrough.js

```diff
@@ -68,7 +71,42 @@ function stepThrough(leftNode, rightNode, comparator, debug=false) {
     }
 
     try {
-      equationStatus = step(equation, symbolName);
+      // Find roots of a equation
+      // Conditions: left side is a mul node, right side is 0
+      // TODO: move polynomial to left if is on the right
+      // TODO: assuming a single symbol in equation
+      // TODO: display all solutions
+      // Returns: An equation in the form of symbol = root
+      if (equation.leftNode.op === '*'
+          && (Node.Type.isConstant(equation.rightNode) && parseFloat(equation.rightNode.value) === 0)) {
```

Comment:
might be worth adding something to `Node.Type.isConstant` that takes an optional parameter of the value we're looking for - would be cleaner and likely helpful elsewhere!